pkg/operator/ceph/csi/spec.go

```diff
@@ -349,11 +356,19 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		if err != nil {
 			return errors.Wrap(err, "failed to load rbdplugin template")
 		}
+		if tp.CSILogRotation {
+			tp.CSILogFolder = "rbdplugin"
+			applyLogrotateSidecar(&rbdPlugin.Spec.Template, "csi-rbd-daemonset-log-collector", LogrotateTemplatePath, tp)
```

Comment:
sorry i am doing back and forth can we name this as `rbd-plugin` what we have today which represents the ds